data-collection/src/test/java/integration/realprojects/ApacheCommonsCliIntegrationTest.java

```diff
@@ -27,12 +29,52 @@ protected String track() {
 		return "src/java/org/apache/commons/cli/Option.java";
 	}
 
+	/*
+	Test the isSubclass boolean
+	 */
+	@Test
+	public void isSubclass() {
+		List<Yes> yesList = session.createQuery("From Yes where (className = :className1 or className = :className2) and project = :project")
+				.setParameter("className1", "org.apache.commons.cli.HelpFormatter")
+				.setParameter("className2", "org.apache.commons.cli.HelpFormatter.StringBufferComparator")
+				.setParameter("project", project)
+				.list();
+
+		Assert.assertEquals(10, yesList.size());
+		for (Yes yes : yesList){
+			switch(yes.getClassName()){
+				case "org.apache.commons.cli.HelpFormatter.StringBufferComparator":
+					Assert.assertTrue(yes.getClassMetrics().isSubclass());
```

Comment:
Maybe not related to this test in particular, but given that everything is complex in this tool, stronger assertions are always better. 

I'm always afraid the tool will confuse both classes. Can we make a simple assertion to make sure that the metrics were collected correctly for the class and the sub-class. A simple assertion on the number of public methods should be enough!